config/config-api/src/com/thoughtworks/go/domain/UnitTestReportGenerator.java

```diff
@@ -151,18 +154,12 @@ private void pumpFileContentIfValid(PrintStream out, File testFile) throws IOExc
     }
 
     private void pumpFileContent(File file, PrintStream out) throws IOException {
-        BufferedReader bufferedReader = null;
         try {
-            bufferedReader = new BufferedReader(new FileReader(file));
-            String line = bufferedReader.readLine();
-            if (!line.contains("<?xml")) { // skip prolog
-                out.println(line);
-            }
-            while ((line = bufferedReader.readLine()) != null) {
-                out.println(line);
-            }
-        } finally {
-            IOUtils.closeQuietly(bufferedReader);
+            String content = FileUtil.readToEnd(file).trim();
+            final Document document = XmlUtils.buildXmlDocument(content);
+            out.println(XmlUtils.asXml(document.getRootElement()));
```

Comment:
If you use `XmlUtils.writeXml(Document, OutputStream)`, this new impl (`XmlUtils.asXml(Element)`) is redundant. Feel free to rename that method if you feel it appropriate.